src/ctfe.h

```diff
@@ -113,6 +113,7 @@ class CTFEExp : public Expression
     static CTFEExp *gotoexp;
 
     static bool isCantExp(Expression *e) { return e && e->op == TOKcantexp; }
+    static bool isGotoExp(Expression *e) { return e && e->op == TOKgoto; }
```

Comment:
Adding `CTFEExp::isGotoExp` is a kind of refactoring. Please separate the change into a commit.